src/views/TaskView.vue

```diff
@@ -4,13 +4,15 @@ import Button from 'primevue/button';
 import Dialog from 'primevue/dialog';
 import InputText from 'primevue/inputtext';
 import TaskService, { Status, type TaskItem } from '@/services/TaskService';
+import TaskTable from '@/components/TaskTable.vue';
 
 const props = defineProps<{
   projectId: string;
   owner?: string;
   status?: Status;
 }>();
 const taskService = new TaskService();
+//const user = new UserService();
```

Comment:
Unused and commented out. If UserService is not used in the near term, remove this line to keep the code clean. I think that you forgot to delete this line of code.